keet: fails to launch

### Nixpkgs version

- Unstable (25.05)

### Describe the bug

`keet` fails to launch.

### Steps to reproduce

Launch `keet`

### Expected behaviour

It works

### Screenshots

_No response_

### Relevant log output

```console
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '../../../../../../boot.js'
Require stack:
- /home/<myUser>/.config/pear/by-dkey/7216eeac5c879a2ca1dcff14bb8c1ad47d188a8c17b13e66f36d354ea8bd3b28/1/by-arch/linux-x64/bin/pear-runtime-app/resources/app/boot.js
- 
    at Module._resolveFilename (node:internal/modules/cjs/loader:1232:15)
    at s._resolveFilename (node:electron/js2c/browser_init:2:124107)
    at Module._load (node:internal/modules/cjs/loader:1058:27)
    at c._load (node:electron/js2c/node_init:2:16955)
    at Module.require (node:internal/modules/cjs/loader:1318:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/home/<myUser>/.config/pear/by-dkey/7216eeac5c879a2ca1dcff14bb8c1ad47d188a8c17b13e66f36d354ea8bd3b28/1/by-arch/linux-x64/bin/pear-runtime-app/resources/app/boot.js:17:5)
    at Module._compile (node:internal/modules/cjs/loader:1484:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1564:10)
    at Module.load (node:internal/modules/cjs/loader:1295:32)
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.71, NixOS, 25.05 (Warbler), 25.05.20250112.ed4a395`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.92.0-dev-pre20250112-38dd196
System type: x86_64-linux
Additional system types: aarch64-linux, arm-linux, armv6l-linux, i686-linux, riscv64-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/<myUser>/.config/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/<myUser>/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/<myUser>/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/<myUser>/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/n6sfn4p1biwwsbqbgr9x8lxrmbpbb57d-lix-2.92.0-dev-pre20250112-38dd196/share`
 - nixpkgs: `/nix/store/2g3q36zra5ix7qhkb8b4ikr5pmmhfm6g-source`

### Notify maintainers

@ImExtends  
---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+bug%22+-label%3A%226.topic%3A+darwin%22+-label%3A%226.topic%3A+nixos%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ke/keet/package.nix
+++ b/pkgs/by-name/ke/keet/package.nix
@@ -6,11 +6,11 @@
 
 let
   pname = "keet";
-  version = "2.2.0";
+  version = "2.4.1";
 
   src = fetchzip {
     url = "https://keet.io/downloads/${version}/Keet-x64.tar.gz";
-    hash = "sha256-Sd2aCUvgxdbCb8MtWMcznX2efmL1h9wLT29GG7t3Gzc=";
+    hash = "sha256-Csr9M9t2brwfK0B8ldXgUQXX6lPH8z2QSkcBgCYSvaE=";
   };
 
   appimageContents = appimageTools.extract {
@@ -26,6 +26,7 @@ appimageTools.wrapType2 {
   extraPkgs =
     pkgs: with pkgs; [
       gtk4
+      graphene
     ];
 
   extraInstallCommands = ''

